dna/__main__.py

```diff
@@ -290,6 +292,81 @@ def evaluate_handler(
         if not os.listdir(plot_dir):
             os.rmdir(plot_dir)
 
+def configure_tuning_parser(parser):
+    parser.add_argument(
+        '--tuning-config-path', type=str, action='store', required=True,
+        help='the directory to read in the tuning config'
+    )
+    configure_evaluate_parser(parser)
+
```

Comment:
This is the change

Reply:
![image](https://user-images.githubusercontent.com/31665361/63218135-05578900-c109-11e9-8a9a-f6a8fc22c256.png)
See the lines numbers?

Reply:
This is not what I am concerned about.